src/compas/data/data.py

```diff
@@ -75,6 +75,10 @@ def __init__(self, name=None):
     def __dtype__(self):
         return "{}/{}".format(".".join(self.__class__.__module__.split(".")[:2]), self.__class__.__name__)
 
+    @classmethod
+    def __cls_dtype__(cls):
```

Comment:
`__clstype__` or `__ctype__` perhaps?

Reply:
ok clstype then, as ctype is a exiting function (C compatible data types)